pkg/cluster/services/kafka.go

```diff
@@ -80,6 +80,7 @@ func (s KafkaService) BuildServiceDetails(serviceName string, namespace string,
 	return &serviceDetails, nil
 }
 
+// Print command to grant service account acccess to the Kafka instance
 func (s KafkaService) PrintAccessCommands(clientID string) {
```

Comment:
All public methods should start with the function name: `// PrintAccessCommand does x`